.github/workflows/lint.yml

```diff
@@ -5,17 +5,16 @@ on:
 jobs:
   lint:
     name: ament_${{ matrix.linter }}
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
           linter: [cppcheck, cpplint, uncrustify, xmllint, copyright]
     steps:
-    - uses: actions/checkout@v1
-    - uses: ros-tooling/setup-ros@0.0.23
-    - uses: ros-tooling/action-ros-lint@0.0.6
+    - uses: actions/checkout@v2
+    - uses: ros-tooling/setup-ros@0.1.4
+    - uses: ros-tooling/action-ros-lint@0.1.3
       with:
         linter: ${{ matrix.linter }}
         package-name: |
             rmw_iceoryx_cpp
-            iceoryx_ros2_bridge
```

Comment:
Why do you remove the bridge?

Reply:
I misunderstood the function of the `iceoryx_ros2_bridge` and will add it back in a week.